### PR TITLE
Extend extension support

### PR DESCRIPTION
**Pull request by mahvaezi**

### CODE REVIEW

**Review by mbastian93 — APPROVED** [review]

File: src/assets/files/questionnaire.js [hunk 1]
```diff
@@ -514,8 +514,56 @@ export default {
...
+                        "valueString": "Option H"
+                    },
+                    {
+                        "valueString": "Option I"
+                    }
+                ],
+                "extension": [{
+                    "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+                    "valueCodeableConcept": {
+                        "CodeableConcept": {
```
Comment (jonathan-reisdorf) [inline src/assets/files/questionnaire.js:553]: This wrapping object is not valid FHIR. In FHIR, the `valueCodeableConcept` object directly contains `coding`

Comment (mbastian93) [reply src/assets/files/questionnaire.js:553]: This was fixed with https://github.com/NUMde/compass-numapp-frontend/commit/eead42f076e1d1021b24ef6296ec84c7e8c0e53e.

Comment (mbastian93) [reply src/assets/files/questionnaire.js:553]: Fixed with 04e90be. Thanks for the hint.

Comment (jonathan-reisdorf) [reply src/assets/files/questionnaire.js:553]: Thank you for the quick fix :)

**Pull request merged**